react/features/base/participants/middleware.js

```diff
@@ -63,20 +63,22 @@ MiddlewareRegistry.register(store => next => action => {
 
     case DOMINANT_SPEAKER_CHANGED: {
         // Ensure the raised hand state is cleared for the dominant speaker.
+
+        const { conference, id } = action.participant;
         const participant = getLocalParticipant(store.getState());
 
         if (participant) {
-            const { id } = action.participant;
-
             store.dispatch(participantUpdated({
+                conference,
                 id,
                 local: participant.id === id,
                 raisedHand: false
             }));
         }
 
-        typeof APP === 'object'
-            && APP.UI.markDominantSpeaker(action.participant.id);
+        if (typeof APP === 'object') {
```

Comment:
you always suggest to use the && syntax, why the change here?

Reply:
Because I'm insecure... I think I changed it in "Coding style" to &&. Then in a later commit I modified the preceding `if` and what thinking: OK, someone's going to see these sequential `if`s and be like "Why is the first one a if and the second one a &&".

Reply:
OK. Will come in "Associate remote participant w/ JitsiConference (_UPDATED)".

Reply:
🤣